amplipi/streams.py

```diff
@@ -1077,21 +1080,27 @@ def connect(self, src):
 
       self.proc = subprocess.Popen(args=lms_args)
       self._connect(src)
+
+      self.metadata_reader.connect()
```

Comment:
This metadata reader needs to be launched as a separate process since connect is expected to return immediately after the LMS stream is started.

The separate process is what will be writing to the metadata file.